kuma/javascript/src/user-provider.jsx

```diff
@@ -108,7 +108,7 @@ export default function UserProvider(props: {
                     }
                 });
             });
-    }, []); // empty array means we only fetch on mount, not on every render
+    }, [ga]); // only fetch on mount, or if `ga` changes
```

Comment:
So, I read about the exhaustive-deps eslint plugin and it sounded like a good thing. But seeing it in action, I've got mixed feelings.

The intent here really is that we only want to run this effect (to fetch user data) once.  ga is a constant, and we know it is not going to change.  The empty array made it clear that the effect would only run once. Having eslint force us to add ga to the array isn't going to change the behavior (since it is constant) but it reduces the clarity of the code.

On the other hand, this is the plugin that would have found the bug that Peter pointed out in code review yesterday in my pr to change menu localization.

Is there a way to configure it to tell it that 'ga' should be exempt? Or do we decide to adopt this lint rule but then supress its warnings on lines like these when we want to explicitly override it? Or do we just go with the simplicity of doing what the linter tells us even when it makes the code less clear?